apim-apk-agent/internal/synchronizer/apis_fetcher.go

```diff
@@ -151,6 +153,47 @@ func FetchAPIsOnEvent(conf *config.Config, apiUUIDList []string, k8sClient clien
 				// Todo: Read the apis.zip and extract the api.zip,deployments.json
 			}
 
+			if err != nil {
+				logger.LoggerMsg.Error("Error while reading zip", err)
+			}
+
+			artifact, decodingError := transformer.DecodeAPIArtifact(data.Resp)
+
+			if decodingError != nil {
+				logger.LoggerSync.Error("Error while decoding the API Project Artifact:", decodingError)
```

Comment:
Use errorf instead of error
 something like this 
```logger.LoggerSync.Errorf("Error while decoding the API Project Artifact: %v", decodingError)```

Follow the same for other places also

Reply:
Changed as suggested